### PR TITLE
Bootfirmware version handling

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-upgrade-alternative.cmd.in [hunk 1]
```diff
@@ -77,7 +77,6 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 			else
 				setenv bootupgrade_available 0;
 				setenv bootupgrade_primary_updated 0;
-				setenv bootfirmware_version "${bootfirmware_version}";
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-upgrade-alternative.cmd.in:80]: Can you extend the commit message saying why they are not needed?

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-upgrade-alternative.cmd.in:80]: I think it was added by mistake, as we obviously re-assign the same value of a variable

**Comment by igoropaniuk** [comment]
@ricardosalveti good to merge?

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**